.github/workflows/scheduled_merge.yml

```diff
@@ -0,0 +1,114 @@
+name: Schedule Merge Staging into Main
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_job:
+        description: 'Execute manually'
+        required: true
+        default: 'yes'
+  schedule:
+    - cron: '0 17 */15 * *'  # execute every 15 days at 17:00 UTC
```

Comment:
I think that every 14 days might be better, this way it lands on the same every-other-week schedule as our TF team meetings, just for the sake of remembering exactly when it is happening!

Reply:
Perfect! I will change this :heavy_check_mark: